x/nft/username/handler.go

```diff
@@ -73,7 +76,7 @@ func (h IssueHandler) Deliver(ctx weave.Context, store weave.KVStore, tx weave.T
 	if err != nil {
 		return res, err
 	}
-
+	res.Tags = append(res.Tags, common.KVPair{Key: []byte(msgTypeTagKey), Value: []byte(newUsernameTagValue)})
```

Comment:
I like this. We should do this for all messages.
Maybe we could auto-perform this with a middleware to tag `msgType` to the `msg.Path()`?
That would be a separate PR in any case.

Reply:
A middleware makes sense when want to go that route. At this stage we have a specific use case we want to cover. I am afraid we could end up with tons of unused tags. Let's think about some other use cases we want to handle first.

Reply:
:+1: